README.md

```diff
@@ -134,28 +139,30 @@ This package provides you with a library of [React](https://reactjs.org/) Compon
 One of the Decathlon Design System goals is to provide guidelines & components to gain in consistency, efficiency & accessibility. The best way to achieve this is together!
 That's why we are on GitHub. We would love contributions from the community _(bug reports, feature requests, suggestions, Pull Requests, whatever you want!)_.
 
-In this monorepo, we use [Yarn](https://yarnpkg.com) & [Lerna](https://github.com/lerna/lerna).
-Read the [Installation Guide](https://yarnpkg.com/en/docs/install) for detailed instructions on how to install Yarn.
+[Yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) are used to manage dependencies and build config across package and [lerna](https://github.com/lerna/lerna/) is used to manage versioning & publishing.
 
-Once you have installed it, please run:
+Run the following to setup your local dev environment:
 
 ```sh
-yarn
-```
+# Install `yarn`, alternatives at https://yarnpkg.com/en/docs/install
+brew install yarn
 
-Then, to develop in watch mode and run all showcases with hot reloads, just run:
+# Clone or fork `vitamin-web`
+git clone git@github.com:Decathlon/vitamin-web.git # or your fork
+cd vitamin-web
 
-```sh
+# install dependencies
+yarn
+
+# start showcases and build sources in watch mode & hot reload
 yarn start
-```
 
-Finally, the different showcases will be launched:
+# build packages
+yarn build
 
-| Showcase       | URL                                     |
```

Comment:
I find this information interesting to provide :)

Reply:
I will put it in CONTRIBUTING.md because it can be confusing with public storybook links already available in this README file :)